src/Pages/MembershipApplication/MembershipForm.js

```diff
@@ -17,71 +17,100 @@ export default function MembershipForm(props) {
   const [major, setMajor] = useState('');
   const [usernameAvailable, setUsernameAvailable] = useState(true);
   const [passwordValid, setPasswordValid] = useState(true);
+  const [submitted, setSubmitted] = useState(false);
+
   const nameFields = [
     {
       label: 'First Name',
       id: 'first-name',
       type: 'text',
-      handleChange: e => setFirstName(e.target.value)
+      minLength: 8,
```

Comment:
First name doesn't have a minimum length. This parameter is never even used and is misleading.